mackup/config.py

```diff
@@ -218,11 +218,10 @@ def _parse_path(self):
                 raise ConfigError("The required 'path' can't be found while"
                                   " the 'file_system' engine is used.")
 
-        # Python 2 and python 3 byte strings are different.
+        # For py2, encode the path with utf8 to bytes string because
+        # there might be non-ascii characters in it.
         if sys.version_info[0] < 3:
-            path = str(path)
-        else:
-            path = path.decode("utf-8")
```

Comment:
under py3, `decode` method is only valid when path is `bytes` type, but path can be either `str` or `bytes` in the original code.